controls/V-72219.rb

```diff
@@ -81,4 +106,18 @@
 Assurance List (CAL), this is a finding."
   tag "fix": "Update the host's firewall settings and/or running services to comply
 with the PPSM CLSA for the site or program and the PPSM CAL."
+
+  if service('firewalld').running?
+    describe firewalld do
+      FIREWALLD_SERVICES_DENY.each do |serv|
+        it { should_not have_service_enabled_in_zone(serv) }
```

Comment:
These zones are completely arbitrary and I have no idea how to do this properly.

`iptables` also needs to be checked since a lot of people still manage that directly.

Reply:
This is testing services in the default zone by default...most of this firewall stuff seems like it will have to have the attributes configured manually depending on the needs/environment of the particular system, so I left all the attributes empty for now.